src/Reference/Task4.java

```diff
@@ -0,0 +1,25 @@
+package Reference;
+
+import java.util.Scanner;
+
+public class Task4 {
+    public static void main(String[] args) {
+
+        Scanner reader = new Scanner(System.in);
+        System.out.println("Enter a fraze: ");
+        String string = reader.nextLine();
+        StringBuffer newStr = new StringBuffer(string);
+
+
+        for(int i = 0; i < string.length(); i++) {
+
+            if(Character.isLowerCase(string.charAt(i))) {
+                newStr.setCharAt(i, Character.toUpperCase(string.charAt(i)));
+            }
+            else if(Character.isUpperCase(string.charAt(i))) {
```

Comment:
In fact, you don't need this validation. If it is not a lower case character, than we have to change it to a lower case one. And that's it. 
So a single validation, which you have above, should be enough.

Reply:
Ok. Understand.